.github/before_merging.md

```diff
@@ -10,5 +10,4 @@ Please make sure you have done these things before asking for approval to finall
 - [ ] Run your application successfully in a [virtual environment](https://realpython.com/python-virtual-environments-a-primer/) and test that other apps still run
 - [ ] Commit and push the files that were generated in the `dash_bio/` subfolder by the above command(s)
 - [ ] Ask for a final review and look for a :dancer:!
-- [ ] Have your dancer? Merge to master!
-
+- [ ] Have your dancer? Merge to master! Merge the default way, without squashing or rebasing.
```

Comment:
Thanks that makes more sense haha I read that section before merging since I wasn't sure, and I just assumed that any type of merging was okay (so i just rebased and merged).... Oops!

Reply:
What about something along the lines of:
```
- [ ] Have your dancer? If you have write access, merge to master! Otherwise, ping a dash-bio member and they will merge it once the tests pass. Make sure to merge the default way, without squashing or rebasing.
```

Reply:
Well, no, I think it's convoluted and unnecessary... the day we review a PR from a contributor who's working on a fork, they won't need to ping us. We will review their PR and once one of us approves it, the latter (one of us) will ping another one of us to get the PR merged in. If we are going to distinguish between contributors who work on a clone vs. on a fork, I think we want a more general paragraph in `CONTRIBUTING.md`.

Reply:
From this point of view, it makes sense! You are right, i think that keeping it simple and leaving it for interpretation (i.e. if an external contributor can't merge, then they know that the message isn't for them).